core/types/receipt_test.go

```diff
@@ -800,11 +867,10 @@ func TestDeriveOptimismBedrockTxReceipts(t *testing.T) {
 func TestDeriveOptimismEcotoneTxReceipts(t *testing.T) {
 	// Ecotone style l1 attributes with baseFeeScalar=2, blobBaseFeeScalar=3, baseFee=1000*1e6, blobBaseFee=10*1e6
 	payload := common.Hex2Bytes("440a5e20000000020000000300000000000004d200000000000004d200000000000004d2000000000000000000000000000000000000000000000000000000003b9aca00000000000000000000000000000000000000000000000000000000000098968000000000000000000000000000000000000000000000000000000000000004d200000000000000000000000000000000000000000000000000000000000004d2")
-	// the parameters we use below are defined in rollup_test.go
-	l1GasPrice := baseFee
-	l1GasUsed := ecotoneGas
-	l1Fee := ecotoneFee
-	txs, receipts := getOptimismTxReceipts(t, payload, l1GasPrice, l1GasUsed, nil /*feeScalar*/, l1Fee)
+	// the parameters we use below are defined in rollup_test.go. Scale the fee scalars by 1e6 to account for the float format.
```

Comment:
```suggestion
	// the parameters we use below are defined in rollup_test.go
```
we removed the scaling again